packages/@icon-magic/distribute/test/fixtures/out/filled-1_filled-24x12.imageset/Contents.json

```diff
@@ -0,0 +1,9 @@
+{
+  "images": [
+    {
+      "idiom": "universal",
+      "scale": "4x",
```

Comment:
We should have other scales too. I can debug into this after you submit the PR, could you create an issue if you're seeing only one here?